islandora_videojs.module

```diff
@@ -70,6 +70,11 @@ function template_preprocess_islandora_videojs(array &$variables) {
   elseif (isset($params['url']) && isset($params['mime'])) {
     $variables['sources'][] = array('url' => $params['url'], 'mime' => $params['mime']);
   }
+  $variables['center_play_button'] = (bool) variable_get('islandora_videojs_center_play_button');
```

Comment:
Default value is not present, but is defined here: https://github.com/Islandora/islandora_videojs/pull/45/files#diff-0cdd19abe14e51fe9e7f061ec72c86d3R23? Similarly, this shouldn't be required to be cast to a bool I think.

Reply:
I agree with the need for a default value.

While the bool casting isn't necessary, is it harmful?